doc/api/dgram.md

```diff
@@ -85,12 +100,18 @@ one interface and will add membership to it. To add membership to every
 available interface, call `addMembership` multiple times, once per interface.
 
 ### socket.address()
+<!-- YAML
+added: v0.1.99
+-->
 
 Returns an object containing the address information for a socket.
 For UDP sockets, this object will contain `address`, `family` and `port`
 properties.
 
 ### socket.bind([port][, address][, callback])
+<!-- YAML
+added: v0.1.99
```

Comment:
IIRC, we have not documenting when an argument or something like a property in an options object is added.